astroquery/mast/core.py

```diff
@@ -1126,8 +1127,26 @@ def query_criteria_async(self, pagesize=None, page=None, **criteria):
         objectname = criteria.pop('objectname', None)
         radius = criteria.pop('radius', 0.2*u.deg)
 
+        if ('obstype' in criteria) and ('intentType' in criteria):
+            warn_string = "Cannot specify both obstype and intentType, "
+            warn_string += "obstype is the deprecated version of intentType and will be ignored."
+            warnings.warn(warn_string, InputWarning)
+            criteria.pop('obstype', None)
+
+        # Temporarily issuing warning about change in behavior
+        # continuing old behavior
         # grabbing the observation type (science vs calibration)
-        obstype = criteria.pop('obstype', 'science')
+        obstype = criteria.pop('obstype', None)
+        if obstype:
+            warn_string = "Criteria obstype argument will disappear in May 2019. "
```

Comment:
This line is never used. To make it more readable I would suggest to define that warning message in one command, if you put the whole message in parenthesis it's easy to continue on the next line).

Reply:
@bsipocz  Ah, I thought you were specifically recommending the decorator which does not allow for a custom warning statement.

I don't know what you mean my using parentheses, but I've changed it so it's only one statement with continuation lines now.